src/kilt/Kilt.ts

```diff
@@ -9,20 +9,32 @@
  * @preferred
  */
 
+import { configuration, configOpts } from '../config/ConfigService'
 import Blockchain from '../blockchain/Blockchain'
 import { clearCache, getCached } from '../blockchainApiConnection'
 
-export function connect(host: string): Promise<Blockchain> {
+export function connect(
+  host: string = configuration.host
+): Promise<Blockchain> {
   return getCached(host)
 }
 
-export async function disconnect(host: string): Promise<void> {
+export async function disconnect(
+  host: string = configuration.host
+): Promise<void> {
   const cached = await getCached(host)
   cached.api.disconnect()
   clearCache()
 }
 
+export function config(configs: Partial<configOpts>): void {
+  const { address, logLevel } = configs
+  if (address) configuration.host = address
+  if (logLevel) configuration.logging = logLevel
```

Comment:
This part illustrates that we'll commit to unnecessary maintenance effort if we hard-code properties into the config service and then also start translating their names. Do we want to add a line here for every new config option?